### ISSUE TITLE
Implement Base.show to make charts viewable in Pluto.jl

### ISSUE THREAD

**Issue by g-gundam** [bhftbootcamp/LightweightCharts.jl#7]
Is your feature request related to a problem? Please describe.
---
> A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

It would be nice if these charts could be viewable inside [Pluto.jl](https://plutojl.org/) notebooks.

Describe the solution you'd like
---
> A clear and concise description of what you want to happen.

This solution was suggested by one of the Pluto.jl maintainers.

```julia
function Base.show(io::IO, mime::MIME"text/html", plt::LightWeightChart)
    write(io, string(plt))
end
```
Link:  [#pluto.jl zulipchat](https://julialang.zulipchat.com/#narrow/stream/243342-pluto.2Ejl/topic/Using.20LightweightCharts.2Ejl.20with.20Pluto.2Ejl/near/443815836)

Describe alternatives you've considered
---
> A clear and concise description of any alternative solutions or features you've considered.

I've used LiveServer.jl to start up an HTTP server to serve the files created by `lwc_save`.  I then used an iframe to display the file inside a Pluto notebook.  It works, but it'd be nice to not have to spin up yet another server.

Additional context
---
> Add any other context or screenshots about the feature request here.

n/a

**Comment by maxfadson** [comment]
@g-gundam, thank you for the suggestion! 
I implemented this in PR #8, you can see the result in these examples:
<details>
  <summary>Jupyter example</summary>
  
<img width="1233" alt="jupyter" src="https://github.com/bhftbootcamp/LightweightCharts.jl/assets/66777881/54a90c27-0072-43d4-b383-a98f777d03de">
  
</details>

<details>
  <summary>Pluto example</summary>
  
<img width="723" alt="pluto" src="https://github.com/bhftbootcamp/LightweightCharts.jl/assets/66777881/763a0aa6-a852-4c0b-80cd-5942c852c52b">
  
</details>

it's really more convenient than using `HTML(string(chart))` :)

**Comment by g-gundam** [comment]
Nice.  Thanks for implementing that.
